command/fileserver/fileserver.go

```diff
@@ -45,6 +48,12 @@ $ step ca certificate 127.0.0.1 localhost.crt localhost.key
 ...
 $ step fileserver --address 127.0.0.1:8443 \
   --cert localhost.crt --key localhost.key /path/to/root
+'''
+
+Start an HTTPS file server on a random port and require client certificates.
+'''
+$ step fileserver --cert localhost.crt --key localhost.key \ 
+  --roots $(step path)/certs/root_ca.crt /path/to/root
```

Comment:
/path/to/root confused me for a while here. Thought it was another root certificate. Can we call this /path/to/web-root or something like that. Given the context, `root` is kinda confusing.

Reply:
Sure